components/Navbar/NavbarMobile/HamburgerMenu.tsx

```diff
@@ -0,0 +1,37 @@
+import clsx from 'clsx'
+import React, { useState } from 'react'
+
+export default function HamburgerMenu(props) {
+  const className =
+    'h-0.5 w-5 bg-white transition-all ease-in-out origin-top-right'
+
+  const aria_label = props.isOpen ? 'Close Menu' : 'Open Menu'
+
+  return (
+    <div
+      className="mr flex h-full w-10 cursor-pointer flex-col items-start justify-center overflow-hidden border-r border-white/25 p-1"
+      onClick={props.clicked}
+      role="button"
+      aria-label={aria_label}
+      aria-expanded={props.isOpen}
+      aria-controls="lesson-navigation-menu"
+    >
+      <div
+        className={clsx(className, 'mb-[5px]', {
```

Comment:
Hey @shaavan, no need to define a className variable, just inline the css classes here.

Reply:
Addressed in ea822132cc3d13887c40e033b9b5d1605f6ed579
Thanks for the suggestion!

I went with extracting class names into variables to decrease repetition. But I understand your point in keeping the code complexity minimal.